com.cubrid.cubridmigration.core.testfragment/.classpath

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/J2SE-1.5"/>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
-	<classpathentry exported="true" kind="lib" path="lib/cglib-nodep-2.1_3.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/hamcrest-core-1.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/hamcrest-library-1.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/jmock-2.5.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/jmock-junit4-2.5.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/jmock-legacy-2.5.1.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
-	<classpathentry exported="true" kind="lib" path="lib/objenesis-1.0.jar" sourcepath="com.cubrid.cubridmigration.core.testfragmentsrc.zip"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17">
```

Comment:
The compiler for the release is jdk 1.8, And eclipse(3.2, indigo) for development are only supports 1.7 or lower.
Please keep the old version until we update eclipse and the build environment.